include/usb/usb_device.h

```diff
@@ -190,6 +191,11 @@ struct usb_cfg_data {
 	usb_interface_config interface_config;
 	/** Callback to be notified on USB connection status change */
 	usb_dc_status_callback cb_usb_status;
+	/** Composite callback */
+	void (*cb_usb_status_composite)(struct usb_cfg_data *cfg,
+					enum usb_dc_status_code cb_status,
+					const u8_t *param);
+
```

Comment:
#ifdef CONFIG_USB_COMPOSITE_DEVICE may be needed for the above added lines.

Reply:
Will fix it in the follow up patches